packages/assets-controllers/src/NftController.test.ts

```diff
@@ -1041,7 +1012,9 @@ describe('NftController', () => {
 
   describe('addNft', () => {
     it('should add NFT and NFT contract', async () => {
-      const { nftController } = setupController();
+      const { nftController } = setupController({
+        getERC721AssetName: jest.fn().mockResolvedValue('Name'),
```

Comment:
Prior to this PR, the real assets contract controller was being called by these tests and hitting the real network. A few additional mocks were needed to keep the tests working.